### DISCUSSION TITLE
Login not possible after a fresh installation

### DISCUSSION THREAD

**Discussion by 1337Rafael1337** [evroon/bracket#1088] [Q&A]
I have installed Bracket on two new servers. On both servers, I am facing the same issue: login is not possible.
![2024-12-21 16_34_26-Bracket und 8 weitere Seiten - Persönlich – Microsoft​ Edge](https://github.com/user-attachments/assets/7574f5d1-76e5-4f46-aa7b-813db2f21b23)
![image](https://github.com/user-attachments/assets/7cebe8b0-07ae-454f-8fd5-11d1973ce304)
![2024-12-21 16_37_57-00-Greenshot und 1 weitere Registerkarte – Explorer](https://github.com/user-attachments/assets/d36ae666-3460-4d75-a178-62050e1e20b5)

**Comment by SwedishChef** [comment]
I have same issue. Please help. THX

**Comment by evroon** [comment]
Could you provide more information, such as:

- operating system
- how is it run, using docker or standalone? 
- did you use the default docker-compose.yml in the case of docker or not?
- what does the browser network tab show?
- did you check any firewalls you might have?

**Comment by 1337Rafael1337** [comment]
I tested it using the latest default docker-compose.yml on an Ubuntu Server 24.04.1 and a Debian 12. Both servers are freshly installed, with no firewalls configured, and have no additional modifications.

![Image](https://github.com/user-attachments/assets/effbefab-25a4-464f-81e8-76c1bc43e20f)

**Comment by evroon** [comment]
Looks like a CORS issue. Have you set the right environment variables as described here when the URL you enter in the browser is not http://localhost:3000?

https://docs.bracketapp.nl/docs/running-bracket/configuration

**Reply by ChiefoftheOwls** [reply]
Hi, I seem to having the same issue. I'm using the http://localhost:3000 address and all the default configurations - still getting the `internal server error` that points to the CORS issue. I've tried the solution in the FAQ to no resolution (i swapped the localhost addresses in the `CORS_ORIGINS` and the `NEXT_PUBLIC_API_BASE_URL` lemme know if i interpreted that wrong). 
I looked at my docker and it seems the back end and postgres container never fully start up (endless restarting).
<img width="627" height="161" alt="Screenshot 2025-11-10 at 12 57 35 PM" src="https://github.com/user-attachments/assets/36510de5-89fb-4f18-940b-cdda0aead49f" />
<img width="979" height="374" alt="Screenshot 2025-11-10 at 12 56 56 PM" src="https://github.com/user-attachments/assets/7267133b-f003-4956-848b-2b1bf0bee451" />

**Comment by SwedishChef** [comment]
I changed CORS_ORIGINS and NEXT_PUBLIC_API_BASE_URL to right URL and now it runs.
@evroon - Many thanks for this hint

I still have '...Permission denied" errors but this seems to have no impact so far.

**Reply by ghost** [reply]
I think those "permission denied" errors are for installing language packs. 
If you speak English, don't worry about it :)

**Comment by evroon** [comment]
Glad it works, CORS seems to be a problem for many users, the docs could make it more clear: https://github.com/evroon/bracket/issues/615

**Comment by fslrshd** [comment]
Hi. I'm having the same problem.

What is the difference between VITE_API_BASE_URL and NEXT_PUBLIC_API_BASE_URL?

Thanks and regards.
fsl
